tensorflow/lite/kernels/kernel_util.h

```diff
@@ -338,6 +338,16 @@ bool IsMobilePlatform();
 // Returns whether there is unspecified dimension in the tensor's dim signature.
 bool HasUnspecifiedDimension(const TfLiteTensor* tensor);
 
+// Returns whether the given type is a quantized type.
+bool IsQuantizedType(TfLiteType type);
+
```

Comment:
This doesn't need to be exposed, it's not used outside of `CheckQuantizationParams`